#!/usr/bin/env python
PACKAGE = "ros_recognizer"

from dynamic_reconfigure.parameter_generator_catkin import *

view_gen = ParameterGenerator()
view_gen.add("show_scene",     bool_t,   0, "View scene input cloud",  True)
view_gen.add("show_model",     bool_t,   0, "View model input cloud",  True)
view_gen.add("show_keypoints", bool_t,   0, "View keypoints",  True)
view_gen.add("show_normals",   bool_t,   0, "View normals",  False)
view_gen.add("show_corrs",     bool_t,   0, "View all correspondences",  False)
view_gen.add("show_clusters",  bool_t,   0, "View only clustered correspondences",  False)
view_gen.add("show_true_hypotheses",   bool_t, 0, "View true hypotheses",  False)
view_gen.add("show_false_hypotheses",  bool_t, 0, "View false hypotheses",  False)

exit(view_gen.generate(PACKAGE, "pcl_recognizer", "Visualizer"))